src/components/workflowEditor/nodes/StaticNode.tsx

```diff
@@ -17,12 +19,20 @@ export interface StaticNodeProps {
     sourceType: string
     regex?: string
     primaryBranchAfterRegex?: string
+    to?: string
+    hideWebhookTippy?: () => void
```

Comment:
is this required?

Reply:
No it was not required here so removed it from the props.